### PR TITLE
Fix/improve lambda metrics extraction process

### PR DESCRIPTION
**Pull request by aryna-yukhnevich**
- alerts updated
image

- unit tests coverage
image

### CODE REVIEW

File: integration_tests/inttest_lib/testing_stand_executor.py [hunk 1]
```diff
@@ -172,6 +172,10 @@ def await_workloads(self):
             runner.await_completion()
 
     def conclude(self):
+        if types.LAMBDA_FUNCTIONS in self.resource_types_to_run:
```
Comment (aryna-yukhnevich) [inline integration_tests/inttest_lib/testing_stand_executor.py:175]: this part will be removed and considered in the corresponding await Lambda method (will be added in a separate PR)

File: src/lib/digest_service/digest_data_extractor.py [hunk 2]
```diff
@@ -145,37 +147,34 @@ class StepFunctionsDigestDataExtractor(BaseDigestDataExtractor):
...
-            f""" AND measure_name = 'execution'  """
-            f""" GROUP BY monitored_environment, resource_name,  error_message  """
-            f""" UNION ALL  """
-            f""" SELECT  monitored_environment, resource_name, count(measure_name) as execution, count(measure_name) as failed,   """
-            f"""         0 as  succeeded, round(max(duration_ms)/60, 2) as execution_time_sec, error_message   """
-            f""" FROM "{self.timestream_db_name}"."{self.timestream_table_name}" WHERE time BETWEEN '{start_time}' AND '{end_time}'  """
-            f""" AND measure_name = 'error'  """
-            f""" GROUP BY monitored_environment, resource_name,  error_message) t  """
-        )
+        query = f"""SELECT '{self.resource_type}' as resource_type, monitored_environment, resource_name, job_run_id
```
Comment (aryna-yukhnevich) [inline src/lib/digest_service/digest_data_extractor.py:167]: Digest part for Lambdas will be additionally improved in a separate PR

File: src/lib/aws/lambda_manager.py [hunk 3]
```diff
@@ -13,39 +13,69 @@
...
+    def IsSuccess(self) -> bool:
+        return self.Status in LambdaManager.LAMBDA_SUCCESS_STATE
 
     @property
-    def IsErrorEvent(self) -> bool:
-        return LambdaManager.MESSAGE_PART_ERROR in self.message
+    def IsFailure(self) -> bool:
+        return self.Status in LambdaManager.LAMBDA_FAILURE_STATE
+
+    def _extract_value(self, pattern: str) -> float:
```
Comment (natmasslove) [inline src/lib/aws/lambda_manager.py:40]: I like this refactoring piece of code :)

**Review by natmasslove — APPROVED** [review]

**Comment by aryna-yukhnevich** [comment]
inttests finished successfully
image
